trpc/router/project.ts

```diff
@@ -522,18 +522,35 @@ export const project = router({
             const parentsToConnect = parent?.filter((pr) => !project.parent.some((p) => p.id === pr.id));
             const parentsToDisconnect = project.parent.filter((p) => !parent?.some((pr) => p.id === pr.id));
 
+            const participantsToConnet = participants?.filter(
+                (pr) => !project.participants.some((p) => p.id === pr.id),
+            );
+            const participantsToDisconnect = project.participants.filter(
```

Comment:
Does this mean that you have to always pass all current participants when you want to update project?

Reply:
Yes, we have some logic for parents several line above and for goal tag